src/app/component/dashboard/customers/customers.component.html

```diff
@@ -91,6 +146,8 @@
       </kendo-grid-column>
       <kendo-grid-column field="mobile" title="{{ language.mobile }}" width="120">
       </kendo-grid-column>
+      <kendo-grid-column field="email" title="{{ language.email }}" width="150">
```

Comment:
Nisam siguran zasto se i ovde prikazuje email i mobile kolona.
Imamo ovo definisano i u 112 liniji.
Da li si prosao nakon spajanja jos jednom aplikaciju i odradio test delova na kojima si radio, da nije doslo do nekih propusta u spajanju, posto si imao konflikte?

Reply:
Ovo je deo kendo-grid-pdf odvojen od linije 112, ako sam dobro razumeo.

Reply:
ok, samo da ne dodje do nekog konflikta
ali ako su to odvojeni delovi, onda je to ok
samo me zanima kako je ovo reseno, ako recimo prikazem 2 kolone, da li ce u export-u biti prikazane samo te dve kolone ili sve one koje su definisane?
cisto baci pogled, ne moras da radis nista u vezi toga u ovoj iteraciji

Reply:
Ovde se prikazuju sve kolone koje su definisana kada se odradi export page to PDF recimo